integration-cli/docker_cli_images_test.go

```diff
@@ -25,14 +25,14 @@ func (s *DockerSuite) TestImagesOrderedByCreationDate(c *check.C) {
 	if err != nil {
 		c.Fatal(err)
 	}
-	time.Sleep(time.Second)
+	time.Sleep(1 * time.Second)
```

Comment:
@cpuguy83 yes. This tests test the order of the ``images`` command. It build 3 images in order and wait 1 seconds between each to be sure the datetime difference is big enough.

I should add a comment on thoses (I though I did but.. seems like I did not… :sweat:).